prediction_market_agent_tooling/deploy/betting_strategy.py

```diff
@@ -0,0 +1,61 @@
+from abc import ABC, ABCMeta, abstractmethod
+
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from prediction_market_agent_tooling.markets.data_models import (
+    Answer,
+    TokenAmountAndDirection,
+)
+from prediction_market_agent_tooling.markets.manifold.manifold import (
+    ManifoldAgentMarket,
+)
+from prediction_market_agent_tooling.markets.omen.omen import OmenAgentMarket
+from prediction_market_agent_tooling.tools.betting_strategies.kelly_criterion import (
+    get_kelly_bet,
+)
+
+
+class BettingStrategy(ABC):
+    @abstractmethod
+    def calculate_bet_amount_and_direction(
+        self, answer: Answer, market: AgentMarket
+    ) -> TokenAmountAndDirection:
+        pass
+
+
+class FixedBetBettingStrategy(BettingStrategy, metaclass=ABCMeta):
```

Comment:
This is no longer abstract class, or is it?
```suggestion
class FixedBetBettingStrategy(BettingStrategy):
```